examples/user_attention/src/main.rs

```diff
@@ -0,0 +1,128 @@
+use eframe::egui::{Button, CentralPanel, Context, UserAttentionType};
+use eframe::{CreationContext, NativeOptions};
+
+use chrono::{DateTime, Duration, Utc};
```

Comment:
using `chrono` is way overkill - you can just use `std::time::SystemTime::now()` and `std::time::Duration`.

(`chrono` has a dependency on an old, broken, version of the `time` crate, so it is more than just extra compile time)